api/db-migrations/000007_log_config_kafka_serialization_format.down.sql

```diff
@@ -0,0 +1,14 @@
+-- Remove 'serialization_format' field in 'kafka_config'
+
+UPDATE router_versions
```

Comment:
Perhaps we can use `jsonb_set()` to simplify this, so we don't have to explicitly specify the fields that wont change: https://www.postgresql.org/docs/11/functions-json.html

Reply:
Updated